Detections/ASimNetworkSessions/SeveralDenyActionsRegistered.yaml

```diff
@@ -0,0 +1,40 @@
+id: D66DD819-F2E6-43F7-8BC1-70BDE8011B39
+name: Several deny actions registered (AsimWebSession)
+description: |
+  'Identifies attack pattern when attacker tries to move, or scan, from resource to resource on the network and creates an incident when a source has more than 1 registered deny action in Azure Firewall.'
```

Comment:
- More than 5.
 - Remove the "in Azure Firewall".
 - Add link to installing ASIM (the full install).

Reply:
Done.